lib/plugins/package/lib/packageService.js

```diff
@@ -67,6 +67,10 @@ module.exports = {
     functionObject.artifact = null; // reset the current artifact
 
     if (funcPackageConfig.artifact) {
+      if (process.env.SLS_DEBUG) {
```

Comment:
I'd prefer a method in the sls framework that is something like `utils.debug()` that handles the env var checking.  However I felt like adding that function to the core is outside the scope of this feature branch/pr.